.github/pull_request_template.md

```diff
@@ -18,4 +18,4 @@ resolves #
 - [ ] I have signed the [CLA](https://docs.getdbt.com/docs/contributor-license-agreements)
 - [ ] I have run this code in development and it appears to resolve the stated issue
 - [ ] This PR includes tests, or tests are not required/relevant for this PR
-- [ ] I have updated the `CHANGELOG.md` and added information about my change to the "dbt-spark next" section.
+- [ ] I have run `changie new` to [create a changelog entry](https://github.com/dbt-labs/dbt-core/blob/main/CONTRIBUTING.md#Adding-CHANGELOG-Entry)
```

Comment:
Since you're updating the contributing doc with this info, should this point to the dbt-spark contributing doc instead?

Reply:
oh good catch will have to fix that on the others as well was still using the core contributing guide as main source since we generally try to send all like information there and just point to it as the source of truth but we are adding the small section in this case. will fix in other repos as well as needed1